pyproject.toml

```
@@ -199,6 +199,7 @@ markers = [
 ]
 log_cli_level = "INFO"
 xfail_strict = true
+filterwarnings = ["error"]
```

```suggestion
filterwarnings = [
    "error",
    "ignore::DeprecationWarning:httpretty.*:",
]
```
You will need something like this atleast due to `httpretty`.